scripts/variantstore/wdl/GvsPrepareBulkImport.wdl

```diff
@@ -35,15 +37,17 @@ workflow GvsPrepareBulkImport {
 # This copies the entirety of the data tables over.  We'll want a different task to make FOFNs for just a sample set
 task GenerateFOFNsFromDataTables {
     input {
-        String project_id
+        String google_project_id
         String workspace_name
+        String workspace_namespace
         String workspace_bucket
-        String samples_table_name = "sample"
-        String sample_id_column_name = "sample_id"
-        String vcf_files_column_name = "hg38_reblocked_v2_vcf"
-        String vcf_index_files_column_name = "hg38_reblocked_v2_vcf_index"
+        String? samples_table_name
+        String? sample_id_column_name
+        String? vcf_files_column_name
+        String? vcf_index_files_column_name
     }
 
+    ## TODO I dont love that we are hardcoding them here and in the python--they need to be params!
```

Comment:
why not just do that now with defaulted params?

Reply:
I want to update this whole python script in my next pr where I also address sample sets